pipeline_objects/feature_engineering_parts/helpers.py

```diff
@@ -1,13 +1,48 @@
 import pandas as pd
 import numpy as np
+from sklearn.impute import KNNImputer
 
 
-def fill_missing_values(df, method = 'knn', **kwargs):
+def fill_missing_values(df, method = 'knn', neighbors, identifier_columns, data_columns, **kwargs):
```

Comment:
lets put these as optional parameters, since these would not be needed if user selects method == 'zero'. You can do `neighbors=None` (or even choose a default value). Same for identifier_columns and data_columns